packages/core/app/layout.tsx

```diff
@@ -9,7 +9,9 @@ import ThirdPartyScripts from 'app/components/ThirdPartyScripts'
 import AnalyticsHandler from 'app/sdk/analytics'
 import ErrorBoundary from 'app/sdk/error/ErrorBoundary'
 import UIProvider from 'app/sdk/ui/UIProvider'
-import { WebFonts } from 'src/customizations/src/GlobalOverrides'
+
+// The path will probably change when overriding fonts in the future
```

Comment:
can you create a TODO here for the overriding fonts and a new task in our backlog? (next 13 epic)

Reply:
I've already created the task, I'll leave the TODO there too 🤝